src/runtime_src/core/pcie/driver/linux/include/xocl_ioctl.h

```diff
@@ -462,13 +466,22 @@ struct drm_xocl_reclock_info {
 	unsigned short ocl_target_freq[DRM_XOCL_NUM_SUPPORTED_CLOCKS];
 };
 
+
+struct drm_xocl_alloc_cma_info {
+	uint64_t page_sz;
+	uint64_t nr_page;
+	bool	 reserve;
+	uint64_t user_addr[DRM_XOCL_CMA_CHUNK_MAX];
```

Comment:
this is not flexible enough for when max number of chunks changes later. can we just do one ioctl for one chunk?